airflow-core/src/airflow/ui/src/i18n/locales/de/README.md

```diff
@@ -62,6 +62,11 @@ Für die Deutsche Übersetzung wurden die folgenden Terme wie folgt übersetzt
 - `Asset Event` --> `Ereignis zu Datenset (Asset)`: Logische Konsequenz der
   Übersetzung von -->"Asset" ohne einen sperrigen Begriff wie
   "Datenssatz-Ereignis" zu erzeugen.
+- `Backfill` --> `Auffüllen`: Der technisch geprägte Term im Programm passt zu
+  dr direkten Übersetzung, auch im Deutschen werden Lücken wieder "aufgefüllt".
```

Comment:
```suggestion
  der direkten Übersetzung, auch im Deutschen werden Lücken wieder "aufgefüllt".
```

Reply:
Doch noch ein "e" in dem Keyboard hngen geblieben :-D